preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>This sentence is a bit vague: </p><blockquote>It is all covered in general English stop words, so we are good.</blockquote><p>I suggest updating to something like: </p><blockquote>All the words we see in the following example are covered in general English stop words, so we will not add any additional stop words. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>